internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -34,10 +34,14 @@ type KopsProvisioner struct {
 	useExistingAWSResources bool
 	resourceUtil            *utils.ResourceUtil
 	logger                  log.FieldLogger
+	store                   model.InstallationDatabaseStoreInterface
 }
 
 // NewKopsProvisioner creates a new KopsProvisioner.
-func NewKopsProvisioner(s3StateStore, owner string, useExistingAWSResources bool, resourceUtil *utils.ResourceUtil, logger log.FieldLogger) *KopsProvisioner {
+// TODO(gsagula): Consider place all this paramaters in a struct for readability.
```

Comment:
good TODO :+1:

Reply:
👍 but there is a typo `Consider place all this` / `Consider place all these`. 
I will fix it!